apps/frontend/src/app/5_pages/AavePage/components/BorrowPositionsList/components/RepayForm/RepayForm.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC, useState } from 'react';
+
+import { t } from 'i18next';
+
+import { Paragraph, ParagraphSize, Tabs, TabType } from '@sovryn/ui';
+
+import { translations } from '../../../../../../../locales/i18n';
+import { RepayWithCollateralForm } from './RepayWithCollateralForm';
+import { RepayWithWalletBalanceForm } from './RepayWithWalletBalanceForm';
+
+type RepayFormProps = {
+  asset: string;
+  onSuccess: () => unknown;
+};
+
+enum RepayWith {
+  BALANCE = 0,
+  COLLATERAL,
+}
+
+export const RepayForm: FC<RepayFormProps> = ({ asset, onSuccess }) => {
+  const [activeTab, setActiveTab] = useState<RepayWith>(RepayWith.BALANCE);
+
+  return (
+    <div className="space-y-2">
+      <Paragraph
+        size={ParagraphSize.small}
+        className="font-medium text-gray-30"
+      >
+        {t(translations.aavePage.repayModal.repayWith)}
+      </Paragraph>
+
+      <Tabs
+        className="flex-grow-0"
+        contentClassName="p-4"
+        index={activeTab}
+        onChange={setActiveTab}
+        items={[
```

Comment:
move this array outside the return in order to not create the array on every render